weather.py

```diff
@@ -56,8 +75,24 @@ def load_data_from_csv(csv_file):
         csv_file: a string representing the file path to a csv file.
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
-    """
-    pass
+    # """
+    with open(csv_file) as csv_file_data:
+        reader = csv.reader(csv_file_data)
+        header = next(reader)
+        return_list = []
+        for line in reader:
+            if line == []:
+                continue
+            else:
+                new_line = []
+                new_line.append(line[0])
+                new_line.append(int(line[1]))
+                new_line.append(int(line[2]))
+                return_list.append(new_line)
```

Comment:
Nice use of the `continue` statement, and storing modified data values to new variables like mentioned above (i.e. using `new_line` rather than modifying `line`)